app/src/spIP/java/org/dhis2/usescases/teiDashboard/ui/TeiDashboardMenu.kt

```diff
@@ -0,0 +1,296 @@
+package org.dhis2.usescases.teiDashboard.ui
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.outlined.Assignment
+import androidx.compose.material.icons.automirrored.outlined.HelpOutline
+import androidx.compose.material.icons.outlined.Cancel
+import androidx.compose.material.icons.outlined.CheckCircle
+import androidx.compose.material.icons.outlined.DeleteForever
+import androidx.compose.material.icons.outlined.DeleteOutline
+import androidx.compose.material.icons.outlined.Flag
+import androidx.compose.material.icons.outlined.LockReset
+import androidx.compose.material.icons.outlined.MoveDown
+import androidx.compose.material.icons.outlined.Share
+import androidx.compose.material.icons.outlined.Sms
+import androidx.compose.material.icons.outlined.Sync
+import androidx.compose.material.icons.outlined.Timeline
+import androidx.compose.material.icons.outlined.Workspaces
+import org.dhis2.R
+import org.dhis2.commons.resources.ResourceManager
+import org.dhis2.usescases.teiDashboard.DashboardEnrollmentModel
+import org.dhis2.usescases.teiDashboard.DashboardViewModel
+import org.dhis2.usescases.teiDashboard.EnrollmentMenuItem
+import org.dhis2.usescases.teiDashboard.EnrollmentMenuItem.ACTIVATE
+import org.dhis2.usescases.teiDashboard.EnrollmentMenuItem.COMPLETE
+import org.dhis2.usescases.teiDashboard.EnrollmentMenuItem.DEACTIVATE
+import org.dhis2.usescases.teiDashboard.EnrollmentMenuItem.DELETE
+import org.dhis2.usescases.teiDashboard.EnrollmentMenuItem.ENROLLMENTS
+import org.dhis2.usescases.teiDashboard.EnrollmentMenuItem.FOLLOW_UP
+import org.dhis2.usescases.teiDashboard.EnrollmentMenuItem.GROUP_BY_STAGE
+import org.dhis2.usescases.teiDashboard.EnrollmentMenuItem.HELP
+import org.dhis2.usescases.teiDashboard.EnrollmentMenuItem.REMOVE
+import org.dhis2.usescases.teiDashboard.EnrollmentMenuItem.SEND_SMS
+import org.dhis2.usescases.teiDashboard.EnrollmentMenuItem.SHARE
+import org.dhis2.usescases.teiDashboard.EnrollmentMenuItem.SYNC
+import org.dhis2.usescases.teiDashboard.EnrollmentMenuItem.TRANSFER
+import org.dhis2.usescases.teiDashboard.EnrollmentMenuItem.VIEW_TIMELINE
+import org.dhis2.usescases.teiDashboard.TeiDashboardContracts.Presenter
+import org.hisp.dhis.android.core.enrollment.EnrollmentStatus
+import org.hisp.dhis.mobile.ui.designsystem.component.menu.MenuItemData
+import org.hisp.dhis.mobile.ui.designsystem.component.menu.MenuItemStyle
+import org.hisp.dhis.mobile.ui.designsystem.component.menu.MenuLeadingElement
+import org.hisp.dhis.mobile.ui.designsystem.theme.SurfaceColor
+import org.hisp.dhis.mobile.ui.designsystem.theme.TextColor
+
+fun getEnrollmentMenuList(
```

Comment:
This is not a good solution for us.

Either change in menus in future versions require change this.